doc/source/parametertree/parametertypes.rst

```diff
@@ -6,68 +6,89 @@ Built-in Parameter Types
 Parameters
 ----------
 
-.. autoclass:: SimpleParameter
```

Comment:
I have somewhat of a preference for keeping these as just the `ClassName` here. The difference is the signature rendered in the docs:

```
class pyqtgraph.parametertree.parameterTypes.SliderParameter(**opts)
```

vs. 

```
class pyqtgraph.parametertree.parameterTypes.slider.SliderParameter(**opts)
```

Since they will almost always be imported via `from pyqtgraph.parametertree import parameterTypes`, the old version may be a little less confusing.

Reply:
Currently the items are ordered by appearance in `PARAM_TYPES`. Do you think they should be ordered alphabetically instead while I'm making changes? Not sure which is preferable.

Reply:
I don't have a strong preference. I guess I'd lean toward alphabetical so it's a little easier to find something when scrolling, but if I know what I'm looking for I'd probably ctrl+f or use the search box anyway.

Reply:
Scratch that -- I thought I took out `set` logic but that means order changes almost every time the script is run... I think I will sort so the order is deterministic and makes clear where new parameters should be added